src/client/containers/SignUp/index.js

```diff
@@ -22,9 +22,26 @@ export default function SignUpContainer() {
       alert("Passwords doesn't match");
       return;
     }
-    const isSignedUp = await signUp({ email, password });
+    const response = await signUp({ email, password });
+    if (response) {
+      try {
+        await fetch('/api/users', {
+          method: 'POST',
+          headers: {
+            'Content-Type': 'application/json',
+          },
+          body: JSON.stringify({
+            uid: response.user.uid,
+            email,
+            username: email,
+          }),
+        });
+        setIsSuccessful(true);
+      } catch {
+        // ignore for now
+      }
```

Comment:
We could abstract this to its own function OR make it part of the function there already exists `signUp` :)